.github/workflows/containers.yaml

```diff
@@ -44,8 +44,8 @@ jobs:
       - name: build and deploy container image to registry
         uses: docker/build-push-action@v5
         with:
-          context: "{{defaultContext}}:containers"
-          file: Dockerfile.${{ matrix.project }}
+          context: "{{defaultContext}}"
```

Comment:
Gotta fix up the "dnsmasq" container cause the file path changed for the `COPY` statements

Reply:
fixed in [0212056](https://github.com/rackerlabs/understack/pull/156/commits/021205665c4fc15f67b265becd40f49dca086925)